newsroom/section_filters/module.py

```diff
@@ -0,0 +1,45 @@
+from quart_babel import lazy_gettext
+
+from newsroom import MONGO_PREFIX
+from newsroom.web.factory import NewsroomWebApp
+from superdesk.core.module import Module
+from superdesk.core.resources import ResourceConfig, MongoIndexOptions, MongoResourceConfig
+
+from .model import SectionFilter
+from .service import SectionFiltersService
+from .views import get_settings_data, section_filters_endpoints
+
+section_filters_config = ResourceConfig(
+    name="section_filters",
+    data_class=SectionFilter,
+    service=SectionFiltersService,
+    mongo=MongoResourceConfig(
+        prefix=MONGO_PREFIX,
+        indexes=[
+            MongoIndexOptions(
+                name="name",
+                keys=[("name", 1)],
+                unique=True,
+                collation={"locale": "en", "strength": 2},
+            )
+        ],
+    ),
+    default_sort=[("name", 1)],
+)
+
+
+def init_module(app: NewsroomWebApp):
+    app.wsgi.settings_app(
+        "section-filters",
+        lazy_gettext("Section Filters"),
+        weight=450,
+        data=get_settings_data,
```

Comment:
I wonder `get_setting_data` is an async function, so why not await this?

Reply:
Because it is awaited when rendering the settings app. Both ways are possible but not sure which one we should stick with. See here in line `36`

https://github.com/superdesk/newsroom-core/blob/ac0d6b6053fb93dbfbb99ef35bd76c26e3a295b0/newsroom/settings/views.py#L33-L39

Reply:
oh right